requirements.prod.in

```
@@ -8,7 +8,7 @@ beautifulsoup4
 bleach
 crispy-bootstrap4
 dj-database-url
-django~=4.2
+django
```

thought: This should be:

```suggestion
django~=5.1
```

so that we stick to the minor version we've upgraded to, but get security fixes, and can decide when we want to upgrade to Django 5.2.

---

I suppose the team maintaining the repo will generally be aware of Django releases and best positioned to decide when the work should scheduled, if we assume that they tend to be difficult to upgrade to (eg due to incompatibility problems). Conversely, keeping this as it is means the automatic updates will ensure these updates reach us, and some may be automatically integrated without issue.

I think I'm neutral on this.

---

My instinct is to go with 5.1 given the key changes that a) introduced features and b) broke our current test setup were introduced in 5.1 (AIUI). I guess in that case if similar things would happen 5.1-5.2, the dependa[bot|becky] PR would have failed tests and we'd know that the upgrade would be nontrivial and could pin in then/do the work.

---

I asked Lucy about whether pinning Django made sense, and Lucy's reply was that we leave it unpinned, so let's do that.